src/main/java/org/apache/fineract/api/BatchApi.java

```diff
@@ -44,10 +51,16 @@ public class BatchApi {
     @Autowired
     private PaymentModeConfiguration paymentModeConfig;
 
+    @Autowired
+    private DateUtil dateUtil;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
     @Value("${application.bucket-name}")
     private String bucketName;
 
-    @GetMapping("/batches")
+    /*@GetMapping("/batches")
```

Comment:
@danishjamal104 dead code. If not needed, can we remove this?

Reply:
resolved